tests/integration-tests/tests/create/test_create/test_create_wrong_os/pcluster.config.ini

```diff
@@ -0,0 +1,19 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = slurm
+master_instance_type = {{ instance }}
+custom_ami = {{ custom_ami }}
+master_root_volume_size = 100
+compute_root_volume_size = 100
+
+[vpc parallelcluster-vpc]
+vpc_id = {{ vpc_id }}
+master_subnet_id = {{ public_subnet_id }}
```

Comment:
If you don't need the compute nodes in a public subnet, you can add them to the private subnet and set use_public_ips to false. This will reduce the number of EIPs needed by the integration tests. Same applies elsewhere

Reply:
I will change it